mapie/conformity_scores/residual_conformity_scores.py

```diff
@@ -132,3 +141,279 @@ def get_estimation_distribution(
         """
         self._check_predicted_data(y_pred)
         return np.multiply(y_pred, np.add(1, conformity_scores))
+
+
+class ConformalResidualFittingScore(ConformityScore):
+    """
+    ConformalResidualFittingScore (CRF) score.
+
+    The signed conformity score = (y - y_pred) / r_pred. r_pred being the
+    predicted residual (|y - y_pred|) of the base estimator.
+    It is calculated by a model that learns to predict these residuals.
+    The learning is done with the log of the residual and we use the
+    exponential of the prediction to avoid negative values.
+
+    The conformity score is symmetrical and allows the calculation of adaptive
+    prediction intervals (taking X into account). It is possible to use it
+    only with split and prefit methods (not with cross methods).
+
+    Warning : if the estimator provided is not fitted a subset of the
+    calibration data will be used to fit the model (50% by default).
```

Comment:
Isn't it 20% ? Because you use the default value of train_test_split

Reply:
Indeed